build.sh

```diff
@@ -24,9 +30,16 @@ installer_url () {
 
 build_6x_7x () {
   declare -r version="${1}";
-  declare -r installer_path="racket-minimal-${version}-x86_64-linux-natipkg.sh";
-  declare -r installer=$(installer_url "${version}" "${installer_path}") || exit "${?}";
-  build "racket" "buildpack-deps:stable" "${installer}" "${version}";
+  declare installer_path;
+  declare installer;
+
+  installer_path="racket-minimal-${version}-x86_64-linux-natipkg.sh";
+  installer=$(installer_url "${version}" "${installer_path}") || exit "${?}";
+  build "racket" "buildpack-deps:stable" "${installer}" "${version}" "${USERNAME}/racket:${version}";
+
+  installer_path="racket-${version}-x86_64-linux-natipkg.sh";
```

Comment:
Instead of reusing the `installer` and `installer_path` variables, let's call these `full_installer` and `full_installer_path`. Plus that way we can keep the variables immutable.

Reply:
Good idea! Done.